lnwallet/chainfee/estimator.go

```diff
@@ -311,11 +311,19 @@ func NewBitcoindEstimator(rpcConfig rpcclient.ConnConfig, feeMode string,
 		return nil, err
 	}
 
-	return &BitcoindEstimator{
+	b := &BitcoindEstimator{
 		fallbackFeePerKW: fallBackFeeRate,
 		bitcoindConn:     chainConn,
 		feeMode:          feeMode,
-	}, nil
+	}
+
+	relayFee := &minRelayFeeManager{
+		fetchNewFee:       b.fetchMinRelayFee,
+		minUpdateInterval: time.Minute * 10,
```

Comment:
Nit: extract `time.Minute * 10` into a constant so it can be reused in the `BtcdEstimator`.